.cspell/balena-words.txt

```diff
@@ -0,0 +1,27 @@
+balena
+balena-api
+balena-base-ui
+balena-build
+balena-builder
+balenahup
+balenalib
+balenistas
+bindmount
+boostrap
+capitano
+contracts
+docsearch
+doxx
+Fincm
+generic
```

Comment:
I wouldn't have thought `generic` or `model` are balena-specific words? :shrug:

Reply:
Cspell detects spellings of the word as wrong but having a subset of the word can be helpful for multiple permutations. So here the error case was probably `genericx86-64-ext`

Reply:
Ahhh. I've not used `cspell` myself, but I have used `codespell` elsewhere, which works in a slightly different way - instead of insisting that _every_ word appears in a dictionary, codespell instead looks for "commonly misspelled words" and suggests a suitable replacement. So please forgive my misunderstanding.

Reply:
(but even then, we only used codespell as a "source of suggestions", and didn't insist that it passed 100% with no errors)